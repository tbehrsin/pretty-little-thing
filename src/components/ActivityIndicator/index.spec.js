import React from 'react';
import { render, act } from '@testing-library/react-native';
import { ActivityIndicator } from './index';
jest.useFakeTimers();

describe('ActivityIndicator', () => {
  test('renders without crashing', () => {
    const { toJSON } = render(<ActivityIndicator />);
    expect(toJSON()).toMatchSnapshot();
  });
});
