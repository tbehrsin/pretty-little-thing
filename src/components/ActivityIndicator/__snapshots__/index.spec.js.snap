// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator renders without crashing 1`] = `
<RNSVGSvgView
  bbHeight={32}
  bbWidth={32}
  focusable={false}
  height={32}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      Object {
        "flex": 0,
        "height": 32,
        "width": 32,
      },
    ]
  }
  width={32}
>
  <RNSVGGroup>
    <RNSVGCircle
      cx={16}
      cy={16}
      opacity={0.4}
      propList={
        Array [
          "stroke",
          "strokeWidth",
        ]
      }
      r={15}
      stroke={4279045389}
      strokeWidth={1}
    />
    <RNSVGPath
      d="M 15.999999999999998 1 A 15 15 0 0 1 31 15.999999999999998"
      propList={
        Array [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={4279045389}
      strokeWidth={1}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
